benchmark/scanner/credsweeper.py

```diff
@@ -26,8 +27,8 @@ def init_scanner(self) -> None:
     def run_scanner(self) -> None:
         self.init_scanner()
         subprocess.call([
-            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--ml_validation",
-            "--save-json", self.output_dir
+            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--save-json",
+            self.output_dir
```

Comment:
```suggestion
            "--jobs", "4", self.output_dir
```

Reply:
Agreed.

Reply:
@babenek do number of threads influence execution time? As we removed `multiprocessing` in favor of `threading` python would not be able to utilize more cores due to the global interpreter lock

In my experience only 2 cores being used even if 16 `--jobs` specified on 16 core processor